spec/support/shared_examples_for_otp_delivery_preference_validation.rb

```diff
@@ -0,0 +1,24 @@
+shared_examples 'an otp delivery preference form' do
+  let(:phone) { '555-555-5000' }
+
+  before do
+    allow_any_instance_of(UpdateUser).to receive(:call)
```

Comment:
@monfresh: Feels weird to stub new 🤔, but I went ahead and changed the test. I used a spy for the phone number capabilities because it handles some other calls to build the error message.